docs/source/user-guide/tabular/tutorials/plot_static_predictions.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+"""
+==============================
+Using Pre-computed Predictions
+==============================
+
+There are several cases in which it's useful to compute the model predictions before using deepchecks.
+
+In some cases the model evaluation can take a long time i.e very large dataset.
```

Comment:
Would probably write this is either as one paragraph or as a bullet list, not as three sentences.